src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,225 +1,83 @@
-            public void doCommand() {
-                setElevatorSpeed(0);
-            }
-        }.withTimeout(EntechCommandBase.DEFAULT_TIMEOUT_SECONDS);
-    }
-    
-    public Command shiftElevatorBack(){
-        return new SingleShotCommand(this) {
-            @Override
-            public void doCommand() {
-                shiftElevatorBackToAllowForRoom();
-            }
-        };
+        elevatorMotor = new WPI_TalonSRX(RobotConstants.CAN.ELEVATOR_MOTOR);
+        elevatorMotorController = new TalonSpeedController(elevatorMotor, MOTOR_SETTINGS.ELEVATOR,true);
+        elevatorMotorController.configure();
+        elevatorMotorController.setDesiredSpeed(0);
+
+        intakeBallSensor = new DigitalInput(RobotConstants.DIGITIAL_INPUT.BALL_SENSOR);
+   
     }
-    
-    @Override
-    public void initialize() {
-        if (intake) {
-            
-            intakeMotor = new WPI_TalonSRX(RobotConstants.CAN.INTAKE_MOTOR);
-            TalonSettings motorSettings = TalonSettingsBuilder.defaults().withCurrentLimits(20, 15, 200).brakeInNeutral()
-                    .withDirections(false, false).noMotorOutputLimits().noMotorStartupRamping().useSpeedControl().build();
 
-            intakeMotorController = new TalonSpeedController(intakeMotor, motorSettings);
-            intakeMotorController.configure();
-            intakeMotor.set(ControlMode.PercentOutput, 0);
-            
-        }
-        
-        if(PNEUMATICS_MOUNTED){
-            
-            deployIntakeSolenoids = new DoubleSolenoid(RobotConstants.CAN.FORWARD, RobotConstants.CAN.REVERSE);
-            
+    public void toggleIntakeArms(){
+        if ( deployIntakeSolenoids.get() == DoubleSolenoid.Value.kForward){
             deployIntakeSolenoids.set(DoubleSolenoid.Value.kReverse);
         }
-        
-        if (elevator) {
-            TalonSettings motorSettings = TalonSettingsBuilder.defaults()
-                    .withCurrentLimits(maxCurrent, maxSustainedCurrent, maxCurrentTime).brakeInNeutral()
-                    .withDirections(false, false).noMotorOutputLimits().noMotorStartupRamping().useSpeedControl().build();
+        else if ( deployIntakeSolenoids.get() == DoubleSolenoid.Value.kForward ){
+            deployIntakeSolenoids.set(DoubleSolenoid.Value.kReverse);
 
-            elevatorMotor = new WPI_TalonSRX(RobotConstants.CAN.ELEVATOR_MOTOR);
-            elevatorMotorController = new TalonSpeedController(elevatorMotor, motorSettings);
-            elevatorMotorController.configure();
-            elevatorMotor.set(ControlMode.PercentOutput, 0);
-        }
-        timer = new Timer();
-        if(BALL_SENSOR){
-            beam = new DigitalInput(RobotConstants.DIGITIAL_INPUT.BALL_SENSOR);
         }
     }
-
+    
     public void deployIntakeArms(){
-        if(PNEUMATICS_MOUNTED){
-            deployIntakeSolenoids.set(DoubleSolenoid.Value.kForward);
-        }
+        deployIntakeSolenoids.set(DoubleSolenoid.Value.kReverse);
     }
```

Comment:
Are you sure we do not need guards for pneumatics?

Reply:
i'm pretty sure-- but lets test it.